src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext/setOffline.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: 'setOffline(offline)'
-excerpt: "Toggles the browser's connectivity on/off."
+excerpt: "Toggles the BrowserContext's connectivity on/off."
 ---
 
-Toggles the browser's connectivity on/off.
+Toggles the BrowserContext's connectivity on/off.
 
-| Parameter | Type    | Description                                                                    |
-| --------- | ------- | ------------------------------------------------------------------------------ |
-| offline   | boolean | Whether to emulate browser being disconnected (`true`) or connected (`false`). |
+| Parameter | Type    | Description                                                                               |
+|-----------|---------|-------------------------------------------------------------------------------------------|
+| offline   | boolean | Whether to emulate the BrowserContext being disconnected (`true`) or connected (`false`). |
```

Comment:
As I suggested before, I'm on the side of saying "the browser context" also here. "the BrowserContext" looks weird :-)

Reply:
Will leave as is for now based on my reply in the other comment.